src/content/en/2022/security.md

```diff
@@ -1,13 +1,15 @@
 ---
 #See https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide#metadata-to-add-at-the-top-of-your-chapters
 title: Security
-#TODO - Review and update chapter description
 description: Security chapter of the 2022 Web Almanac covering Transport Layer Security, content inclusion (CSP, Feature Policy, SRI), web defense mechanisms (tackling XSS, XS-Leaks), and drivers of security mechanism adoptions.
 authors: [SaptakS, feross, lirantal]
 reviewers: [kushaldas, tomvangoethem, nrllh, clarkio]
 analysts: [VictorLeP, vikvanderlinden, GJFR]
```

Comment:
@SaptakS / @feross / @lirantal can you confirm all contributors are listed, and in descending order of contribution?

If some didn't end up having time to contribute, then please remove them to be fair to others that did.
If we've missed anyone, please add them.

Reply:
@tunetheweb:
- @clarkio should be listed in the authors
- I think he took over ferros's chapters as he couldn't make it